daprdocs/config.toml

```diff
@@ -10,6 +10,12 @@ enableGitInfo = true
 # Language Configuration
 languageCode = "en-us"
 
+[server]
+[[server.headers]]
+  for = '/**.html'
```

Comment:
In my local testing, `/**.html` did not work. I had to use `/**` for the response headers to return. However, when I did that, Chrome developer extensions showed something interesting for `.js` files.  I'm not sure its its a real issue or not.

![image](https://user-images.githubusercontent.com/714039/135538968-9a12777d-b5f0-448b-bde2-1c6eda7cc516.png)

Reply:
My config snippet was:

```toml
[server]
[[server.headers]]
  for = '/**/'
  [server.headers.values]
    Content-Security-Policy = 'script-src localhost:1313'
    Referrer-Policy = 'strict-origin-when-cross-origin'
    X-Content-Type-Options = 'nosniff'
    X-Frame-Options = 'DENY'
    X-XSS-Protection = '1; mode=block'
```